tests/unit/report.spec.ts

```diff
@@ -0,0 +1,15 @@
+const normalizeNewline = require('normalize-newline');
+const read = require('read-file-relative').readSync;
```

Comment:
Have you already looked in to using the Node native 'fs' and 'path' modules? These native modules have type definitions in place. 

I don't see that much of a benefit for this particular use-case for using this dependency. If you'd really want to minimize code, you could create a separate (typed) utility function for this particular function.

```ts
import fs from 'fs';
import path from 'path';
// ...
let expected = fs.readFileSync(path.resolve(__dirname, '../data/report'), { encoding: 'utf8' });
```

* readFileSync: https://nodejs.org/dist/latest-v12.x/docs/api/fs.html#fs_fs_readfilesync_path_options
* __dirname: https://nodejs.org/api/modules.html#modules_dirname
* path.resolve: https://nodejs.org/api/path.html#path_path_resolve_paths

Reply:
Removed the dependency on `read-file-relative`, moved report loading and normalizing to the utility folder.